TAC/Meetings/2022/2022-10-06/CCC-TAC-Minutes-2022-10-06.md

```diff
@@ -0,0 +1,149 @@
+   * Salus (TSM) Github - https://github.com/rivosinc/salus
+   * Platform Security Model Specification Github - https://github.com/riscv-non-isa/riscv-platform-security-model
+* ACTION: [Lily] recommendation is to make sure Keystone is participating
+
+**Meeting Minute Approval**
+
+Proposed: That the minutes of the September 22, 2022 meeting of the Technical Advisory Council of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
+   * Quorum was reached and the minutes were approved by the voting members of the TAC. KT to amend if there was a missing action or recommendation from the MPC presentation.
+
+**IETF Hackathon**
+
+* https://www.ietf.org/how/runningcode/hackathons/115-hackathon/
+* Veraison project asking for 1K to cover travel
+* Outreach: Not about sponsoring the event, supporting a project, TAC to decide
+* Dan Middleton (DM) asked for clarification on Veraison's participation, volunteered to take the idea to the Attestation SIG
+* DT proposed a change in budget to cover some approved travel expenses, same approval process as a hardware request
+* PROPOSED: Projects can request up to $2k per year for travel to hackathons or conferences for open source projects to promote and/or test their project. Requests are approved by the TAC on a case by case basis.
+   * Deferred due to not having quorum, will continue vote via email or the next meeting
+
+**Common Test Infrastructure**
+
+* Only heard back from 3 projects, only one finding it useful so far, wait to check with Occlum
+* If no additional interest, defer the proposal for common test, projects can request later if needed
+
+**Outreach Committee**
+
+* CCC Landscape and Solutions Map - Nick Vidal (NV)
+   * Like CNCF, not just landscape but solutions using services
+   * DT noted that commercial offerings should not be included on the website
+   * ACTION: [Kurt/Nick] add this discussion to the Governing Board agenda, Nick to start the Solutions Map discussion via email
+* Wikipedia - NV discussed an edit-a-thon for creating the CCC page
+   * Seeking volunteers to help define the draft - https://docs.google.com/document/d/1G-bvnXLg4l1r5mMoMRQH7a6bufwGW_7w45bKcy_zdOE/edit#heading=h.lbcyx38ifcd4
+   * Wikipedia process linked in the email thread
+
+**Common Terminology Whitepaper Working Session**
+
+* DT reviewed the current open PRs for terminology
+	* Remove confidential container term #141
+	  * https://github.com/confidential-computing/governance/pull/141
+	  * Closed, instead use #142
+	* Fix confidential container definition #142
+	  * https://github.com/confidential-computing/governance/pull/142
+	  * Will merge by next meeting if no objections
+	* Proposal to include attestation in definition of Confidential Computing #135
+	  https://github.com/confidential-computing/governance/pull/135
```

Comment:
Do we need to capture an explicit TAC vote on this (c.f. board decision on this verbiage 2022/10/19 based on communicated TAC vote).

Reply:
"The TAC voted to approve the change and take it to the board for ratification."